deploy/templates/nstemplatetiers/appstudio-env/ns_env.yaml

```diff
@@ -25,7 +25,7 @@ objects:
   kind: ServiceAccount
   metadata:
     namespace: ${SPACE_NAME}-env
-    name: namespace-manager
+    name: ${SERVICE_ACCOUNT_NAME}
```

Comment:
i think it is fine to hard code the name here and in the tier.yaml. Also because unless I'm missing something, you are not populating this variable `SERVICE_ACCOUNT_NAME` right now.